pymc3/tests/test_examples.py

```diff
@@ -212,7 +213,7 @@ def test_disaster_model_missing(self):
             # Use slice sampler for means (other variables auto-selected)
             step = pm.Slice([model.early_mean_log__, model.late_mean_log__])
             tr = pm.sample(500, tune=50, start=start, step=step, chains=2)
-            pm.summary(tr)
+            az.summary(tr)
```

Comment:
Just had a revelation that now that we are updating this, we could also start using `return_inferencedata=True` when possible, here for example we could do:

```
    tr = pm.sample(..., return_inferencedata=True)
az.summary(tr)
```

once we are using an inferencedata it is not necessary to call `az.summary` from inside a model context. Does this sound useful to prepare people for `return_inferencedata=True` becoming the default?

Reply:
It would definitely be useful! Although once this does become the default, maybe we'll have to remove the kwarg, which would be cumbersome 🤔

Reply:
We should change all tests to explicitly pass `return_inferencedata` in one PR.

Only then we can swap the default, which I think we should do with v4. After that we can, but don't need to take out the `return_inferencedata=True` appearances in one PR.